products/numpy.md

```diff
@@ -0,0 +1,49 @@
+---
+title: NumPy
+permalink: /numpy
+category: framework
+releasePolicyLink: https://numpy.org/neps/nep-0029-deprecation_policy.html
+versionCommand: python -c "import numpy; print(numpy.__version__)"
+auto:
+-   git: https://github.com/numpy/numpy
+-   regex: ^v(?<major>[0-9]+)\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)$
+releases:
+-   releaseCycle: "1.14"
+    eol: 2020-01-07
+    releaseDate: 2018-01-06
+
+-   releaseCycle: "1.15"
+    eol: 2020-07-23
+    releaseDate: 2018-07-23
+
+-   releaseCycle: "1.16"
+    eol: 2021-01-13
+    releaseDate: 2019-01-13
```

Comment:
Please add `latest` version (including patch version) on all of the release cycles.

Reply:
Done. I've also added NumPy 1.23, as it can be found on https://pypi.org/project/numpy/#history. Since the version wasn't mentioned in the deprecation document, I assume that they don't yet have an eol date for this version.